test/jdk/tools/jpackage/share/AppImagePackageTest.java

```diff
@@ -61,4 +65,37 @@ public static void test() {
             cmd.removeArgumentWithValue("--input");
         }).addBundleDesktopIntegrationVerifier(false).run();
     }
+
+    @Test
+    public static void testEmpty() {
+        final String name = "EmptyAppImagePackageTest";
+        final String imageName = name + (TKit.isOSX() ? ".app" : "");
+        Path appImageDir = TKit.workDir().resolve(imageName);
+
+        try {
+            Path dir = Files.createDirectory(appImageDir);
+            Files.createDirectory(dir.resolve("bin"));
+            Path libdir = Files.createDirectory(dir.resolve("lib"));
+            Path readme = Files.createFile(libdir.resolve("README"));
+            try (BufferedWriter bw = Files.newBufferedWriter(readme)) {
+                bw.write("This is some arbitrary ext for the README file\n");
```

Comment:
All these lines can be replaced with a single call

`TKit.createTextFile(List.of("This is some arbitrary text for the README file"));`

This statement will add log statements, so no manual logging is required. Also there is no need for try/catch block